frontend/kesaseteli/employer/src/components/application/steps/step2/error-notification/EmploymentsErrorNotification.tsx

```diff
@@ -0,0 +1,69 @@
+import { $Notification } from 'kesaseteli/employer/components/application/steps/step2/error-notification/EmploymentsErrorNotification.sc';
+import { useTranslation } from 'next-i18next';
+import React from 'react';
+import { FieldError, useFormContext } from 'react-hook-form';
+import {
+  $Grid,
+  $GridCell,
+} from 'shared/components/forms/section/FormSection.sc';
+import Application from 'shared/types/employer-application';
+import Employment from 'shared/types/employment';
+
+import EmployeeErrorNotification from './EmployeeErrorNotification';
+
+const EmploymentsErrorNotification: React.FC = () => {
+  const { t } = useTranslation();
+  const {
+    getValues,
+    formState: { isValid, errors, isSubmitted },
+  } = useFormContext<Application>();
+
+  const getEmploymentId = React.useCallback(
+    (index: number): Employment['id'] =>
+      getValues(`summer_vouchers.${index}.id`),
+    [getValues]
+  );
+
+  if (!errors || isValid || !isSubmitted) {
+    return null;
+  }
+  const employmentsErrors = errors.summer_vouchers || [];
+  const employmentErrorEntries = Array.isArray(employmentsErrors)
+    ? employmentsErrors.map((employmentErrors, index) => ({
+        index,
+        errors: Object.entries(employmentErrors ?? {}).map(
+          ([field, error]) => ({
+            field: field as keyof Employment,
+            errorType: error?.type || 'required',
```

Comment:
It seems that it would never fallback to that value because error object should not be undefined, I think I added it for the typescript only.  Also the errorType needs some of the react-hook-form error type value:
`errorType: LiteralUnion<"required" | "min" | "max" | "maxLength" | "minLength" | "pattern" | "validate" | "valueAsNumber" | "valueAsDate" | "value" | "setValueAs" | ..., string>
`